python/GangaDirac/Lib/Splitters/OfflineGangaDiracSplitter.py

```diff
@@ -189,6 +187,13 @@ def calculateSiteSEMapping(file_replicas, uniqueSE, site_to_SE_mapping, SE_to_si
     while len(site_to_SE_mapping) != maps_size:
         time.sleep(0.1)
 
+    for iSE in site_to_SE_mapping.keys():
```

Comment:
Actually removing keys is potentially good for setups with many sites as the lookup is faster